jupyterlab/labapp.py

```diff
@@ -14,6 +14,8 @@
 from jupyter_server._version import version_info as jpserver_version_info
 from jupyter_server.serverapp import flags
 from jupyter_server.utils import url_path_join as ujoin
+from jupyter_server import _tz as tz
+from jupyter_server.transutils import _i18n
```

Comment:
JupyterLab is not using the `jupyter_server` translation tool. Could you remove this?